#fieldtype1 fieldname1
#
int8 IR_1
int8 IR_2
float32 US_1